app/controllers/api/base_controller/authentication.rb

```diff
@@ -49,10 +49,11 @@ def authorize_user_group(user_obj)
       end
 
       def validate_user_identity(user_obj)
-        @user_validation_service ||= UserValidationService.new(self)
-        missing_feature = @user_validation_service.missing_user_features(user_obj)
+        missing_feature = User.missing_user_features(user_obj)
         if missing_feature
-          raise AuthenticationError, "Invalid User #{user_obj.userid} specified, User's #{missing_feature} is missing"
+          msg = "Invalid User #{user_obj.userid} specified, User's #{missing_feature} is missing"
+          $log.warn(msg)
```

Comment:
why the extra $log.warn,  all API raised errors, including authentication are in api.log.